gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1770,7 +1837,7 @@ def test_{{ service.name|snake_case }}_rest_lro_client():
     # Ensure that we have a api-core operations client.
     assert isinstance(
         transport.operations_client,
-        operations_v1.OperationsClient,
+        operations_v1.AbstractOperationsClient,
```

Comment:
Why was this changed?

Reply:
This was at the request of software-dov.

Reply:
Regarding related PRs:  I'm not sure.  I believe other people are making these changes and releasing them, I'm not sure of the timeline or PR numbers.  The change to noxfile.py is small, but I won't know exactly how to do it until I know what the actual release numbers for showcase and api-core are.